static/lang/re-en.json

```diff
@@ -3884,7 +3884,24 @@
                 "Third": "3rd-Rank"
             },
             "Spellshape": {
-                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[(@item.level)d4[fire]] damage. If the creature summoned has the @Damage[(@item.level)d4[acid]]{acid}, @Damage[(@item.level)d4[cold]]{cold}, @Damage[(@item.level)d4[electricity]]{electricity}, @Damage[(@item.level)d4[sonic]]{sonic}, or @Damage[(@item.level)d4[spirit]]{spirit} trait, you can deal damage of that type instead."
+                "BondConservation": "By efficiently and carefully manipulating the arcane energies unleashed by your bonded item, you can conserve just enough power to cast another spell, though this second spell is slightly weaker. You gain an extra use of Drain Bonded Item. You can use it to cast a spell 2 or more ranks lower than the previous spell, and must use it before the end of your next turn or you lose it.",
+                "ConcealSpellWitch": "Through sheer mental effort, you can simplify the incantations and gestures needed to spellcast, leaving them barely noticeable. If the next action you use is to Cast a Spell, the spell gains the subtle trait, hiding the shining runes, sparks of magic, and other manifestations that would usually give away your spellcasting. The trait hides only the spell's spellcasting actions and manifestations, not its effects, so an observer might still see a ray streak out from you or see you vanish into thin air.",
```

Comment:
No need to have multiple Notes for the same thing. Strip out the flavour and just get the mechanics, which are the same among classes.

Reply:
Fair.
I was trying to keep to the spirit of the text, but in hindsight that was a lot of work. I can trim these down.

Reply:
Yeah, as a general rule, Notes and effect descriptions should have their flavour stripped out. Especially important with notes considering how much space they take in the chat otherwise.